copernicusmarine/core_functions/documentation_utils.py

```diff
@@ -229,7 +229,7 @@
         "Save the downloaded data with the given file name (under the output "
         "directory)."
     ),
-    "FILE_FORMAT_HELP": "Format of the downloaded dataset. Default to NetCDF '.nc'.",
+    "FILE_FORMAT_HELP": "Format of the downloaded dataset. If not set or None, defaults to NetCDF '.nc' if possible or CSV '.csv' for sparse datasets.",  # noqa: E501
```

Comment:
```suggestion
    "FILE_FORMAT_HELP": "Format of the downloaded dataset. If not set (or None), defaults to NetCDF '.nc' if possible or CSV '.csv' for sparse datasets.",  # noqa: E501
```
I don't really like the phrasing of this sentence. How do we call the datasets that are non-sparse? I don't know if I like this 'if possible'.

Reply:
Also, should we add a list of the accepted formats here?

Reply:
Even if it is for gridded and sparse data, I don't know.